src/pages/documentation/mobile-core/configuration/index.md

```diff
@@ -80,16 +80,6 @@ Xamarin
 
 <Tabs query="platform=xamarin&task=update"/>
 
-## Clearing programmatic updates to the configuration
-
-<InlineAlert variant="info" slots="text"/>
-
-This API is only available in Android and iOS (AEP 3.x).
```

Comment:
@praveek should we keep this here for Android 1.x?

Reply:
Yes. We should keep it only for Android.

Reply:
Yes, i apologize i didn't update the ticket after our discussion. @calebk1 We should only remove the iOS note, since this was available for Android in the older versions.

Reply:
I'll just change it to "This API is only available in Android" for this version then!